src/System.Linq.Parallel/tests/WithCancellationTests.cs

```diff
@@ -33,6 +33,24 @@ public static void WithCancellation_Multiple_CancelableToken(Labeled<ParallelQue
             Assert.Throws<InvalidOperationException>(() => labeled.Item.WithCancellation(token).WithCancellation(new CancellationTokenSource().Token));
         }
 
+        [Theory]
+        [MemberData(nameof(Sources.Ranges), new[] { 16 }, MemberType = typeof(Sources))]
```

Comment:
Same question as on the previous commit.  All this is doing is validating that WithCancellation returns successfully, right?  That doesn't depend on all these different sources.